saml-core/src/main/java/org/keycloak/saml/processing/core/parsers/saml/metadata/SAMLAssertionParser.java

```diff
@@ -0,0 +1,15 @@
+package org.keycloak.saml.processing.core.parsers.saml.metadata;
+
+public class SAMLAssertionParser extends SAMLAttributeTypeParser {
```

Comment:
Remove and use parser from `org.keycloak.saml.processing.core.parsers.saml.assertion`